api/src/main/resources/org/openmrs/api/db/hibernate/Encounter.hbm.xml

```diff
@@ -59,7 +59,7 @@
 			<column name="form_id" />
 		</many-to-one>
 		<!-- bi-directional one-to-many association to Order -->
-		<set name="orders" inverse="true">
+		<set name="orders" inverse="true" where="order_group_id is null">
```

Comment:
May be i missed one of the design calls, why do you have to ignore orders in the encounter that belong to an order group?